examples/index.html

```diff
@@ -48,7 +48,6 @@ <h3 class="font-size-small">In Text:</h3><br/>
   <script>
     var blurredLocation = new BlurredLocation();
```

Comment:
We don't need to show everything in this example, but we may eventually add more examples to show more obscure cases. For now the example should show a set of common functions in an easy-to-read scenario.